pkg/controller/job-controller.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/minio/minio-go/v7/pkg/set"
 	"github.com/minio/operator/pkg/apis/job.min.io/v1alpha1"
 	miniov2 "github.com/minio/operator/pkg/apis/minio.min.io/v2"
+	v1alpha12 "github.com/minio/operator/pkg/apis/sts.min.io/v1alpha1"
```

Comment:
Why `v1alpha12` and not just `v1alpha1` ;)

Reply:
already used for `github.com/minio/operator/pkg/apis/job.min.io/v1alpha1` @shtripat

Reply:
ah ok.